stoplight.js

```diff
@@ -0,0 +1,18 @@
+var red = $("#red")
+var yellow = $("#yellow")
+var green = $("#green.bulb")
+
+red.on("click", turnRed)
+yellow.on("click", turnYellow)
+green.click(function(){
+  green.toggleClass("active")});
+
+
+function turnRed(){
+  red.addClass("active")
+}
+function turnYellow(){
+  yellow.addClass("active");
+  setInterval(function(){
+    yellow.removeClass("active");}, 2000);
+  }
```

Comment:
How would you turn a light off if its already on? How would you make it so only one light can be turned on at a time?